grapher/rule.go

```diff
@@ -17,6 +18,7 @@ import (
 )
 
 const graphOp = "graph"
+const goUnitType = "GoPackage"
```

Comment:
Maybe this is something that a toolchain passes to the Register function? (whether or not it wants to receive all source units at once in the graph phase or to have separate graph phases for each source unit?)